workspaces/ai-integrations/plugins/catalog-backend-module-model-catalog/src/clients/ModelCatalogGenerator.ts

```diff
@@ -122,6 +122,20 @@ export function GenerateModelResourceEntities(
         `component:${modelServer.name}`,
       );
     }
+
+    // Handle any annotations present on the model resource
+    if (model.annotations !== undefined) {
+      // Add the techdocs annotation to the resource if present
+      if (model.annotations.TechDocs !== '') {
```

Comment:
should we trim for spaces and then check for the empty string

Reply:
also, should we enforce that the annotation is a http/https URL

Reply:
> should we trim for spaces and then check for the empty string

Yes, we should do that given the free form nature of the input textbox in RHOAI

> also, should we enforce that the annotation is a http/https URL

IMO, this should be handled in the bridge if we do this.

Reply:
ok then ... I'm working on PR to make the polling interval / push to backstage configurable ... I'll include it there